test/acceptance/pages.py

```diff
@@ -75,6 +74,29 @@ def submit_response(self, response_text):
         self.submit()
         EmptyPromise(lambda: self.has_submitted, 'Response is completed').fulfill()
 
```

Comment:
@awaisdar001 can we refactor this line to reflect the new selector. https://github.com/edx/edx-ora2/blob/aj/tnl1755-fix-latex-button-on-ORA/test/acceptance/pages.py#L73